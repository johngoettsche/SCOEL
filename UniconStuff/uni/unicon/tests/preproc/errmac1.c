/* test error message: too few macro arguments */

#define m(a,b) a b
m(3)
