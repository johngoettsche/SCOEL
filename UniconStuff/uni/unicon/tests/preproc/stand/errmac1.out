pp: File errmac1.c; Line 4: too few arguments for macro call to m
